ambuda/consts.py

```diff
@@ -20,5 +20,6 @@
     ],
     "anye": [
         "bodhicaryavatara",
+        "catuhsloki",
```

Comment:
`sloki` -> `shloki`. We have a few informal processes (e.g. how to romanize devanagari URLs here) that need more definition, and we can formalize them more in Q4